src/main/java/racingcar/GameManager.java

```diff
@@ -0,0 +1,51 @@
+package racingcar;
+
+import racingcar.condition.RandomCondition;
+import racingcar.domain.Car;
+import racingcar.view.InputView;
+import racingcar.view.OutputView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GameManager {
+    private static final String MESSAGE_NOT_EXIST_CAR = "생성된 자동차가 없습니다.";
+    OutputView outputView = new OutputView();
+    InputView inputView = new InputView(outputView);
+
+    public void run() {
+        List<Car> cars = new ArrayList<>();
+        List<String> carNames = inputView.getCarName();
+        makeNewCars(cars, carNames);
```

Comment:
현재로직은 기존의 cars에 새로운 car들을 생성해서 넣어주는 기능이니 makeNewCars보단 addNewCars의 역할을 하고 있네요!
메서드명과 어울리는 역할을 수행하려면 
```suggestion
        List<String> carNames = inputView.getCarName();
        List<Car> cars = makeNewCars(cars, carNames);
```
이런식으로도 수정할 수 있겠네요!